elements/pl-file-preview/pl_file_preview_utils.py

```diff
@@ -0,0 +1,21 @@
+from typing import List
+
+
+def order_files(
+    submitted_files: List[dict], required_file_names: List[str]
```

Comment:
Lowercase dict annotations ara python 3.10 only feature! Also missing type of this dict.